src/internal/minikubetestenv/deploy.go

```diff
@@ -85,20 +87,25 @@ func helmChartLocalPath(t testing.TB) string {
 }
 
 func GetPachAddress(t testing.TB) *grpcutil.PachdAddress {
-	if computedPachAddress == nil {
-		cfg, err := config.Read(true, true)
-		require.NoError(t, err)
-		_, context, err := cfg.ActiveContext(false)
-		require.NoError(t, err)
-		computedPachAddress, err = client.GetUserMachineAddr(context)
-		require.NoError(t, err)
-		if computedPachAddress == nil {
-			copy := grpcutil.DefaultPachdAddress
-			computedPachAddress = &copy
+	addr := grpcutil.DefaultPachdAddress
+	if *hostOverride != "" {
+		addr.Host = *hostOverride
+	} else if exposedServiceType() == "NodePort" {
+		in := new(bytes.Buffer)
+		in.WriteString("minikube ip")
+		out := new(bytes.Buffer)
+		cmd := exec.Command("/bin/bash")
```

Comment:
Any reason we need the indirection through the shell here?  If we do need it, you can also `strings.NewReader("minikube ip")` to save one line of code vs bytes.buffer ;)